File: core/src/ru/myitschool/cubegame/skills/targeting/TilePos.java
package ru.myitschool.cubegame.skills.targeting;

import ru.myitschool.cubegame.tiles.DungeonTile;

/**
 * Created by Voyager on 25.02.2018.
 */
public class TilePos {

    private int x;
    private int y;
    private DungeonTile tile;

    public TilePos(int x, int y, DungeonTile tile) {
        this.x = x;
        this.y = y;
        this.tile = tile;
    }

    public int getX() {
        return x;
    }

    public int getY() {
        return y;
    }

    public DungeonTile getTile() {
        return tile;
    }
}

File: core/src/com/wolg_vlad/dcrawler/entities/skills/targeting/TilePos.kt
package com.wolg_vlad.dcrawler.entities.skills.targeting

import com.wolg_vlad.dcrawler.ui.tiles.DungeonTile

/**
 * Class for tile with  coordinates. Used by displayers to interact with [DynamicTileLayer]
 */
data class TilePos(val x: Int, val y: Int, val tile: DungeonTile)